app/import_functions.go

```diff
@@ -1224,7 +1224,7 @@ func (a *App) importAttachment(c *request.Context, data *AttachmentImportData, p
 
 	// Go over existing files in the post and see if there already exists a file with the same name, size and hash. If so - skip it
 	if post.Id != "" {
-		oldFiles, err := a.GetFileInfosForPost(post.Id, true, false)
+		oldFiles, _, err := a.GetFileInfosForPost(post.Id, true, false)
```

Comment:
@isacikgoz does this seem accurate? As in, is this used for importing data directly into the database for new cloud instances that are importing existing data? And if so, I'm thinking we'll need either a new function or another param so that we don't accidentally limit importing data here.

Reply:
This is being used in the import path but I couldn't see if anything being limited on the import path. But in theory I'd like to have a separate method with calculated `firstInaccessibleFileTime` as it can cause error during imports and we may still want to continue to import process.

Reply:
@neallred I'll add a couple of new funcs, which "ignore" cloud limits. And we can use those here for import.